config/loader.go

```diff
@@ -1,56 +1,85 @@
 package config
 
 import (
-	"fmt"
-	"log"
-	"strings"
-
 	"github.com/knadh/koanf"
 	"github.com/knadh/koanf/parsers/yaml"
 	"github.com/knadh/koanf/providers/env"
 	"github.com/knadh/koanf/providers/file"
 	"github.com/knadh/koanf/providers/structs"
+	"log"
```

Comment:
sort import packages using `goimports`, see [this](https://bariseser.medium.com/useful-go-command-1-goimports-7a9eaa7869a3).